tests/VObject/VCardConverterTest.php

```diff
@@ -38,9 +38,9 @@ public function testConvert30to40(): void
 TEL;PREF=1;TYPE=HOME:+1 555 666 777
 ITEM1.TEL:+1 444 555 666
 ITEM1.X-ABLABEL:CustomLabel
-PHOTO;TYPE=HOME:data:image/jpeg;base64,Zm9v
-PHOTO:data:image/gif;base64,Zm9v
-PHOTO;X-PARAM=FOO:data:image/png;base64,Zm9v
+PHOTO;TYPE=HOME:data:image/jpeg;base64\\,Zm9v
+PHOTO:data:image/gif;base64\\,Zm9v
```

Comment:
I think we should not change the existing test but add a new one so we keep backwards compatibility...?

Reply:
I don't know whether existing clients might also send malformed requests?

Or existing scripts relied on this example

Reply:
Just so there is no confusion: vobject will continue to parse an unescaped comma here as before, this is just a change in the test data. If you prefer to keep it malformed, no problem. I thought it would be better if the actual test samples are properly escaped as to not confuse the reader. Escaping of the commas in the test data I did only for this reason and is not required concerning my code changes.

Reply:
Hello, will it help to get this merged if I revert all unnecessary changes in the tests?